portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path

```diff
@@ -785,6 +785,16 @@
 	<td>//a[contains(@class,'btn-sm btn-primary') and text()='Done']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>SELECT_TRANSLATION_LANGUAGE</td>
+	<td>//div[contains(@class,'col') and h3='${key_title}']//button[contains(@class,'dropdown-toggle')]//*[name()='svg'][contains(@class,'lexicon-icon-${key_currentLocale}')]</td>
```

Comment:
Try to use elements or attributes that are more related to the name of the path. ie: class = "translation-manager" or "translation-language".

I don't have the sample portlet available to view right now but something like `//*[contains(@class,'select-translation-language')]//button[contains(@class,'dropdown-toggle')]`

If `select-translation-language` doesn't exist then we can rename the path name to something that's more closely associated to the available html attributes.